### PR TITLE
Added oms-events integration

### PR DESCRIPTION
**Pull request by blacksph3re**
I added the routes and puppet declarations to integrate the [oms-events](https://github.com/AEGEE/oms-events) microservice into the virtual machine

### CODE REVIEW

**Comment by blacksph3re** [comment]
I couldn't find out how to install gulp via puppet, so it's still required to run `npm install -g gulp`

**Comment by linuxbandit** [comment]
Please modify the readme to show the port :)

Also since you're modifying I'll tell you how to do puppet, just copy lines 18-23 of site.pp (it's how the package forever is installed)

**Comment by blacksph3re** [comment]
Don't accept yet, maybe I destroyed something

**Comment by linuxbandit** [comment]
you didn't modify the inside, I think it can cause problems :v

**Comment by linuxbandit** [comment]
Also as per my original comment, the line where you install gulp and the line in the readme where you describe the port forwarding and folder sync

**Comment by blacksph3re** [comment]
Okay, everything should be fine. As I was struggling with my services a little I also learned vagrant and added a provisioner which autostarts all our services in a tmux.

**Comment by linuxbandit** [comment]
not feeling well today, will have a look tomorrow or over the weekend (also Saturday it's A-Berlin's birthday and you should come, maafaka)

File: Vagrantfile [hunk 1]
```diff
@@ -49,8 +52,12 @@ Vagrant.configure(2) do |config|
...
-  config.vm.provision :shell, :inline => "sudo service mongod start"
-  config.vm.provision :shell, :inline => "sudo service redis-server start"
+  config.vm.provision :shell, :inline => "sudo npm install gulp -g"
 
+  #sometimes puppet does not start it and i cannot redeclare it in the manifest so here it is
+  config.vm.provision :shell, run: "always", :inline => "sudo service mongod start"
+  config.vm.provision :shell, run: "always", :inline => "sudo service redis-server start"
+  #Will start tmux sessions inside the machine running the services
+  #You can check their console output with tmux attach -t [core/profiles/events]
+  config.vm.provision :shell, run: "always", path: "runall.sh", privileged: false
```
Comment (linuxbandit) [inline Vagrantfile]: please put it inside the folder scripts/

File: README.md [hunk 2]
```diff
@@ -58,13 +47,16 @@ I tried to make the Vagrantfile as readable as possible but it's worth repeating
 |---|---|---|---|
 |8800|API|oms-core|8080|
 |8801|consumer|oms-profiles-module|8081|
+|---|API|oms-events|8082|
```
Comment (linuxbandit) [inline README.md:50]: no forwarding? useful if you want to test via curl (provided you snoop the token)... not mandatory though :)

File: README.md [hunk 3]
```diff
@@ -15,18 +15,7 @@ Simple LDAP server with Vagrant and Puppet as proof-of-concept of these technolo
...
-one must ssh into the machine, change directory, and manually invoke node:
-
-* vagrant ssh
-* cd /srv/oms-core/lib
-* node server.js &
-* cd /srv/oms-profiles-module/lib
-* node server.js &
-
-they will start normally, but when invoked with the upstart script nope.. (maybe the fault is of "forever")
+* Maybe you will have to create the folders oms-events, oms-cron and oms-token-master in /ignore
```
Comment (linuxbandit) [inline README.md:18]: cool, can you do it please? the empty folders are not tracked so I added a .delme empty file in there, just do the same for the new folders

File: README.md (hunk 2 above)
Comment (blacksph3re) [inline README.md:50]: have a reverse-proxy on :8803/api =)
In the end we should set up one with nginx anyway, so I did it already.

File: README.md (hunk 3 above)
Comment (blacksph3re) [inline README.md:18]: folders are present

**Pull request closed without merging**